python/mxnet/symbol/symbol.py

```diff
@@ -1437,6 +1437,41 @@ def _gen_atomic_symbol(self):
         return Symbol(handle)
 
 
+    def optimizeFor(self, backend, ctx=None, args=None, **kwargs):
```

Comment:
should be optimize_for, since we don't do camelCase in api functions

Reply:
changed function name